config.ts

```diff
@@ -2,11 +2,27 @@ const config = {
   QRAPHQL_SERVER: 'https://serve.giveth.io/graphql',
   APP_URL: 'https://typescript.giveth.io/',
   REPORT_ISSUE_URL: 'https://github.com/Giveth/giveth-io-typescript/issues/new',
-  GIVeconomy_URL: 'https://liquidity-mining-dapp.vercel.app/'
+  GIVeconomy_URL: 'https://liquidity-mining-dapp.vercel.app/',
+  OUR_SECRET: 'our_secret_message_string',
```

Comment:
I know we are not using OUR_SECRET at the moment but as soon as we start doing mutations we weill. Let's move this to the env file

Reply:
I copied that from the env file